GiwazipClient/Views/SettingViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SettingViewController.swift
+//  GiwazipClient
+//
+//  Created by 김민택 on 2023/02/15.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+class SettingViewController: UICollectionViewController {
+
+    // MARK: - Property
+
+    private let settingList = [["📝 고객 정보 수정"],
+                               ["📢 공지사항", "📄 이용약관", "🧑🏼‍💻 개발자 정보", "☕️ 오픈소스 라이브러리", "📬 고객센터", "📱 버전 정보"],
+                               ["🔚 시공 마감하기"]]
+
+    private enum Section: CaseIterable {
+        case userSetting
+        case appSetting
+        case roomSetting
+    }
+
+    private lazy var dataSource: UICollectionViewDiffableDataSource<Section, String> = {
+        let cellRegistration = UICollectionView.CellRegistration<UICollectionViewListCell, String>
+        { cell, indexPath, setting in
+            var content = cell.defaultContentConfiguration()
+            content.text = setting
+
+            if indexPath.section == 2 {
+                content.textProperties.color = .red
+            }
+
+            cell.contentConfiguration = content
+            cell.accessories = [.disclosureIndicator()]
+        }
+
+        return UICollectionViewDiffableDataSource<Section, String>(collectionView: collectionView)
+        { (collectionView, indexPath, setting) in
+            collectionView.dequeueConfiguredReusableCell(using: cellRegistration,
+                                                         for: indexPath,
+                                                         item: setting)
+        }
```

Comment:
해당 부분의 41번째 라인은 아래와 같이도 작성할 수 있습니다만, Swift 최신 문법을 따라 클로저 내부의 반환 타입을 생략하여 현재와 같이 작성했습니다.
해당 부분이 모호하다고 판단 되신다면, 반환 타입을 포함하도록 변경하겠습니다.
```suggestion
        return UICollectionViewDiffableDataSource<Section, String>(collectionView: collectionView)
        { (collectionView, indexPath, setting) -> UICollectionViewCell? in
            collectionView.dequeueConfiguredReusableCell(using: cellRegistration,
                                                         for: indexPath,
                                                         item: setting)
        }
```

Reply:
> 해당 부분의 41번째 라인은 아래와 같이도 작성할 수 있습니다만, Swift 최신 문법을 따라 클로저 내부의 반환 타입을 생략하여 현재와 같이 작성했습니다. 해당 부분이 모호하다고 판단 되신다면, 반환 타입을 포함하도록 변경하겠습니다.

아래가 suggest인가요? suggest를 사용하니 뭔가 아래꺼로 수정했다는 느낌이 들어서 확인차 묻습니다.
현재 반영되어 있는 코드는 위 코드로 되어있습니다. 코멘트 확인 후 답변 부탁드립니다.

Reply:
> > 해당 부분의 41번째 라인은 아래와 같이도 작성할 수 있습니다만, Swift 최신 문법을 따라 클로저 내부의 반환 타입을 생략하여 현재와 같이 작성했습니다. 해당 부분이 모호하다고 판단 되신다면, 반환 타입을 포함하도록 변경하겠습니다.
> 
> 아래가 suggest인가요? suggest를 사용하니 뭔가 아래꺼로 수정했다는 느낌이 들어서 확인차 묻습니다. 현재 반영되어 있는 코드는 위 코드로 되어있습니다. 코멘트 확인 후 답변 부탁드립니다.

말씀하신대로 현재는 `-> UICollectionViewListCell?` 부분이 반영되어 있지 않습니다.
해당 부분이 생략된 것이 모호하다고 생각된다면 추가해도 좋을 것 같습니다.

Reply:
> 말씀하신대로 현재는 `-> UICollectionViewListCell?` 부분이 반영되어 있지 않습니다. 해당 부분이 생략된 것이 모호하다고 생각된다면 추가해도 좋을 것 같습니다.

그대로 해도 괜찮을 것 같습니다.